os/include/tinyara/pm/pm.h

```diff
@@ -303,9 +303,14 @@ enum pm_state_e {
 	PM_COUNT,
 };
 
-struct pm_wakeup_timer_s {
-	bool use_timer;
-	uint32_t timer_interval;
+enum pm_timer_type_e {
+	USE_PM_WAKEUP_TIMER = 1,
+	USE_PM_LOCK_TIMER = 2,
+};
+
+struct pm_timer_s {
```

Comment:
Lets keep the struct name as pm_wakeup_timer_s, it is decided by Mr.Chang.

Reply:
The timer usage is not restricted to waking up now so I generalized the name

Reply:
Okay, make sense to me~